src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -251,6 +247,7 @@ export class DotEditContentHtmlService {
 
     private setEditContentletStyles(): void {
         const doc = this.getEditPageDocument();
+        const robotoFontElement = this.dotDOMHtmlUtilService.createLinkElement(GOOGLE_FONTS);
         const style = doc.createElement('style');
```

Comment:
Please add a comments here so we update the ```dotDOMHtmlUtilService``` to add a method to create this style there.